showcase/src/assets/examples/dialogs/predefined-dialogs.ts

```diff
@@ -0,0 +1,88 @@
+import { Component, Inject } from "@angular/core";
+import { MatDialog } from "@angular/material/dialog";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+import {
+	StarkAlertDialogComponent,
+	StarkAlertDialogContent,
+	StarkAlertDialogResult,
+	StarkConfirmDialogComponent,
+	StarkConfirmDialogContent,
+	StarkConfirmDialogResult,
+	StarkPromptDialogComponent,
+	StarkPromptDialogContent,
+	StarkPromptDialogResult
+} from "@nationalbankbelgium/stark-ui";
+
+@Component({
+	selector: "demo-dialogs",
+	templateUrl: "./demo-dialogs.component.html"
+})
+export class DemoDialogsComponent {
+	public dialogStatus: string;
+
+	public constructor(@Inject(STARK_LOGGING_SERVICE) public logger: StarkLoggingService, public dialogService: MatDialog) {}
+
+	public showAlert(): void {
+		this.dialogService
+			.open<StarkAlertDialogComponent, StarkAlertDialogContent, StarkAlertDialogResult>(StarkAlertDialogComponent, {
+				data: {
+					title: "This is an alert title",
+					textContent: "You can specify some description text in here.",
+					ok: "Got it!"
+				},
+				ariaLabel: "Alert Dialog Demo"
+			})
+			.afterClosed()
+			.subscribe((result: StarkAlertDialogResult) => {
+				if (result === "ok") {
+					this.dialogStatus = "Alert dialog closed with value '" + result + "'";
```

Comment:
This is not a blocker for met but I really like using template strings. 
IE 
```
`Alert dialog closed with value '${result}'` 
```

Reply:
Indeed, it is an old habit I should change :p

Reply:
I've fixed this also ;)